src/Transports/MassTransit.AmazonSqsTransport/AmazonSqsTransport/Middleware/AmazonSqsMessageReceiver.cs

```diff
@@ -62,15 +62,19 @@ async Task Consume()
 
             SetReady();
 
+            Task Handle(Message message, CancellationToken cancellationToken)
+            {
+                var lockContext = new AmazonSqsReceiveLockContext(_context.InputAddress, message, _receiveSettings, _client);
```

Comment:
So, this is created, but it has a Task inside of it. How do we ensure that this Task is properly managed/disposed of - is that taken care of even if the pool doesn't actually handle the message due to being canceled, or something?